### PR TITLE
Introduced v1 of Unbound Web Stats - Text based

### PR DESCRIPTION
**Pull request by juched78**
Base work for Unbound stats page with script and ASP page.

Use "unbound_stats.sh install" to install and make modifications for page to show.
Use "unbound_stats.sh unisntall" to remove.
Use "unbound_stats.sh generate" to update stats.

Runs every hour at 59 minutes after the hour.

Also added version header to adblock gen script as suggested.

### CODE REVIEW

**Comment by juched78** [comment]
Thanks for the feedback. I really don’t expect people to run this script as it is just a helper script which is enabled or disabled by the unbound_manager one which is why I didn’t add a menu.  That output is just in case someone runs it.

Good point on the data being lost on reboot. I will add in a call to generate them again at startup.

Currently the data will never grow as it is just that text. But I do plan to do the ntp graphs for cache hit % over time.  Was thinking to use SQLite like you did.  Thoughts?

File: unboundstats_www.asp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+p{
+font-weight: bolder;
+}
+
+</style>
+<script language="JavaScript" type="text/javascript" src="/js/jquery.js"></script>
+<script language="JavaScript" type="text/javascript" src="/state.js"></script>
+<script language="JavaScript" type="text/javascript" src="/general.js"></script>
+<script language="JavaScript" type="text/javascript" src="/popup.js"></script>
+<script language=JavaScript" type="text/javascript" src="/help.js"></script>
```
Comment (pasha) [inline unboundstats_www.asp]: You're missing a `"` in `language=JavaScript"`

Comment (juched78) [reply unboundstats_www.asp]: Thank you, pushed a fix.

**Comment by jackyaz** [comment]
Are you happy for this to be merged now?

**Comment by juched78** [comment]
> Are you happy for this to be merged now?

Probably best to coordinate with @Martineau as the sites list (and perhaps permlist and blocklist too).  If we push too early, then it will break the ad-block script.

**Comment by jackyaz** [comment]
Script will be maintained on @juched78 repo, no PR necessary

**Pull request closed without merging**